wasm/module.go

```diff
@@ -68,44 +69,16 @@ type Module struct {
 }
 
 func (m *Module) GetSections() []Section {
-	var sections []Section
-	if m.Types != nil {
-		sections = append(sections, m.Types)
-	}
-	if m.Import != nil {
-		sections = append(sections, m.Import)
-	}
-	if m.Function != nil {
-		sections = append(sections, m.Function)
-	}
-	if m.Table != nil {
-		sections = append(sections, m.Table)
-	}
-	if m.Memory != nil {
-		sections = append(sections, m.Memory)
-	}
-	if m.Global != nil {
-		sections = append(sections, m.Global)
-	}
-	if m.Export != nil {
-		sections = append(sections, m.Export)
-	}
-	if m.Start != nil {
-		sections = append(sections, m.Start)
-	}
-	if m.Elements != nil {
-		sections = append(sections, m.Elements)
-	}
-	if m.Code != nil {
-		sections = append(sections, m.Code)
-	}
-	if m.Data != nil {
-		sections = append(sections, m.Data)
-	}
+	return m.Sections
+}
+
+func (m *Module) GetCustom(name string) *SectionCustom {
```

Comment:
`s/GetCustom/Custom/` ?

Reply:
I was also thinking to rename `Other` to `Custom` and leave `GetCustom` as-is